build/protobuf.mk

```diff
@@ -30,7 +30,7 @@ PROTOC_PLUGIN   := $(LOCAL_BIN)/protoc-gen-$(PLUGIN_SUFFIX)
 GOGOPROTO_PROTO := $(GOGO_PROTOBUF_PATH)/gogoproto/gogo.proto
 
 COREOS_PATH := $(GITHUB_ROOT)/coreos
-COREOS_RAFT_PROTOS := $(addprefix $(COREOS_PATH)/etcd/raft/, $(sort $(shell git -C $(COREOS_PATH)/etcd/raft ls-files --exclude-standard --cached --others -- '*.proto')))
+COREOS_RAFT_PROTOS := $(sort $(shell find $(COREOS_PATH)/etcd/raft -type f -name '*.proto'))
```

Comment:
OOC, any reason for the change to find? Performance, consistency, or clarity?

Reply:
The source tarballs strip the Git checkout to reduce the size of the tarball. (The `.git` directory on my machine is 1.6GB, which is _crazy_.) So `git ls-files` doesn't work, but `find` does. This `git ls-files`wasn't an issue before because `make build` didn't call `make protobuf`, but the removal of embedded.go means every `make build` invocation checks to make sure the protobufs are up-to-date.